wbb/__init__.py

```diff
@@ -98,24 +98,25 @@ async def load_sudoers():
             SUDOERS.add(user_id)
 
 
-loop = asyncio.get_event_loop()
+loop = asyncio.get_event_loop_policy().get_event_loop()
 loop.run_until_complete(load_sudoers())
 
 if not HEROKU:
     app2 = Client(
-        "userbot",
+        name="userbot",
         phone_number=PHONE_NUMBER,
         api_id=API_ID,
         api_hash=API_HASH,
+        in_memory=True,
```

Comment:
Remove in memory from userbot client, otherwise people have to enter phone number and verify using OTP everytime they restart the bot

Reply:
done